docs/documentation/functions/Install-VCFCertificate.md

```diff
@@ -289,6 +310,22 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -uploadPrivateKey
+
+Option to upload a custom private key when performing the ESXi host certificate replacement (only valid for VCF 5.2 and later).

```

Comment:
```suggestion
Option to upload a custom private key when performing the ESXi host certificate replacement. Supported on VMware Cloud Foundation 5.2 or later.
```

Reply:
addressed in commit e6a7844965249faebca841b4da521bef3b7b40ba